iocore/net/quic/QUICFrame.h

```diff
@@ -97,6 +99,7 @@ class QUICStreamFrame : public QUICFrame
   virtual size_t store(uint8_t *buf, size_t *len, size_t limit) const override;
   virtual int debug_msg(char *msg, size_t msg_len) const override;
   virtual void parse(const uint8_t *buf, size_t len) override;
+  void parse(IOBufferBlock *block);
```

Comment:
Is this used? It might make sense in the future, but please remove it if it's currently not used.

Reply:
Ops, missed that.